src/check-for-ami.js

```diff
@@ -39,7 +39,7 @@ module.exports = async function (ec2, ami) {
   }
 
   if (result.data.Images[0].ImageId === ami) {
-    return true;
+    return result.data.Images[0];
```

Comment:
@jhford as we discussed, I changed what `check-for-ami` method returns, apparently it didn't break anything outside amiset, but it would be great if you could check it too, please :)